compose/docker-compose.yaml

```diff
@@ -44,6 +44,7 @@ services:
     ports:
```

Comment:
I'm reverting this docker-compose setup. I can't get score uploads to work without using `network_mode: "host"` because the hostname doesn't match the presigned url. I thought this [config var](https://github.com/overture-stack/score/blob/89bb8339a04ea3acf2e9bcd16f745586ee8bf30b/score-server/src/main/resources/application.yml#L41) might be the work around for this but it didn't work.

Reply:
:disappointed: but ok